apps/web/playwright/team/team-invitation.e2e.ts

```diff
@@ -0,0 +1,121 @@
+import { expect } from "@playwright/test";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+
+import { test } from "../lib/fixtures";
+import { expectInvitationEmailToBeReceived } from "./expects";
+
+test.describe.configure({ mode: "parallel" });
+
+test.afterEach(async ({ users, emails, clipboard }) => {
+  clipboard.reset();
+  await users.deleteAll();
+  emails?.deleteAll();
+});
+
+test.describe("Team", () => {
+  test("Invitation (non verified)", async ({ browser, page, users, emails, clipboard }) => {
+    const teamOwner = await users.create(undefined, { hasTeam: true });
+    const { team } = await teamOwner.getFirstTeam();
+    await teamOwner.apiLogin();
+    await page.goto(`/settings/teams/${team.id}/members`);
+    await page.waitForLoadState("networkidle");
+
+    await test.step("To the team by email (external user)", async () => {
+      const invitedUserEmail = `rick_${Date.now()}@domain-${Date.now()}.com`;
+      await page.locator('button:text("Add")').click();
+      await page.locator('input[name="inviteUser"]').fill(invitedUserEmail);
+      await page.locator('button:text("Send Invite")').click();
+      await page.waitForLoadState("networkidle");
+      const inviteLink = await expectInvitationEmailToBeReceived(
+        page,
+        emails,
+        invitedUserEmail,
+        `${team.name}'s admin invited you to join the team ${team.name} on Cal.com`,
+        "signup?token"
+      );
+
+      //Check newly invited member exists and is pending
+      await expect(
+        page.locator(`[data-testid="email-${invitedUserEmail.replace("@", "")}-pending"]`)
+      ).toHaveCount(1);
+
+      // eslint-disable-next-line playwright/no-conditional-in-test
+      if (!inviteLink) return null;
+
+      // Follow invite link to new window
+      const context = await browser.newContext();
+      const newPage = await context.newPage();
+      await newPage.goto(inviteLink);
+      await newPage.waitForLoadState("networkidle");
+
+      // Check required fields
+      await newPage.locator("button[type=submit]").click();
+      await expect(newPage.locator(".text-red-700")).toHaveCount(3);
```

Comment:
I prefer if we were a bit more specific on this locator - text-red-700 is easily added else where and would break this test